.gitpod.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM gitpod/workspace-full
+
+USER gitpod
+
+RUN bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh && \
+    sdk install java 17.0.3-tem && \
```

Comment:
Does not seem right to hard-code a patch release here, in a way opaque to Dependabot.

Reply:
Good point.
Any idea to explore in order to fix/improve this ?

Reply:
Sorry, not offhand.

Reply:
Thank you.
I will run this problem with the people from Infra and see if they have an idea. 

In the mean time, I propose to  flag this as a technical debt.